homeassistant/components/squeezebox/strings.json

```
@@ -103,5 +103,16 @@
         "unit_of_measurement": "[%key:component::squeezebox::entity::sensor::player_count::unit_of_measurement%]"
       }
     }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "title": "LMS Configuration.",
```

There should be no period at then end of the title. Also, is the acronym `LMS` something most users would understand? How about something like this instead:

```suggestion
        "title": "Lyrion Music Server Options",
```

---

I'll remove the .  Actually LMS is better.  Until recently (and still in many people's heads) it was known as Logitech Media Server.  The choice of Lyrion Music Server was deliberate so that we could carry the LMS acroynym forwards.